vnext/src/Libraries/AppTheme/AppThemeTypes.ts

```diff
@@ -18,12 +18,6 @@ export interface IHighContrastColors {
   WindowTextColor: string;
 }
 
-export type AppThemeTypes = 'light' | 'dark';
-
-export interface IAppThemeChangedEvent {
```

Comment:
You'll probably need to run "yarn api" from the repo root after making TS API changes